reinforcement_learning/rl_bin_packing_newsvendor_vehicle_routing_ray_customEnv/rl_bin_packing_ray_custom.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<blockquote>This notebook shows an example of how to use reinforcement learning technique to address an online and stochastic bin packing problem. In the classic version of bin packing, we are given items of different sizes and need to pack them into as few bins as possible. This problem is NP-hard. In the online stochastic version, items arrive one at a time and item sizes are drawn from an unknown distribution. The task is to find a feasible packing that minimizes the number of bins used to pack all of the items that arrive within the time horizon.</blockquote><p>This notebook shows an example of how to use reinforcement learning <strong>to solve the</strong> <strong>online stochastic bin packing</strong> problem. In the classic version of bin packing, we are given items of different sizes and need to pack them into as few bins as possible. In the online stochastic version, items arrive one at a time and item sizes are drawn from an unknown distribution. The task is to find a feasible packing that minimizes the number of bins used to pack all of the items that arrive within the time horizon.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/934/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>